case-lib/lib.sh

```diff
@@ -364,3 +364,25 @@ logger_disabled()
         return 1
     fi
 }
+
+print_module_params()
+{
+    echo "---------> Printing module parameters <----------"
+    # this is snd_intel_dspcfg for dsp_driver
+    for param in /sys/module/snd_intel_dspcfg/parameters/*
+    do
+        if [[ -f $param ]]; then
+           printf "$param = "'%s\n' "$(cat $param)"
+        fi
+    done
+
+    # for all the *sof* modules
+    for param in /sys/module/*sof*/parameters/*
+    do
+        if [[ -f $param ]]; then
+           printf "$param = "'%s\n' "$(cat $param)"
+        fi
+    done
```

Comment:
I think you can replace this loop with just one line:
```shell
grep ^ /sys/module/*sof*/parameters/*
```

Reply:
this is very short and great! Thank you for the recommendation